src/webui/www/private/scripts/statistics.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * MIT License
+ * Copyright (C) 2025 Thomas Piccirello <thomas@piccirello.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+"use strict";
+
+window.qBittorrent ??= {};
+window.qBittorrent.Statistics ??= (() => {
```

Comment:
I've opted to place this logic in a separate file for better separation. With browser caching, I don't foresee much impact from this.

If we don't want a separate file for this logic, I would propose maintaining the separate `Statistics` namespace and moving it to an existing file, e.g. client.js.

Reply:
Have you consider using a `class`? I suppose the `save` and `render` as class functions would fit well. And it won't strictly require a separate file.
ps. I don't have strong opinion about it being in client.js or its own separate file.

Reply:
>I didn't consider a class - I tried to follow the same structure as our other JS.

Reason I'm talking about it is that we recently introduced [custom element](https://github.com/qbittorrent/qBittorrent/pull/22768) and the statistics window looks like a good use case for custom element. I imagine the code would look more organized.
This is just for discussion, not requesting for change though.

Reply:
Maybe it's just because custom elements are a new concept to me, but they seem like overkill for what's effectively a bunch of text elements.